### ISSUE TITLE
Accelerometer simulator not accessible

### ISSUE THREAD

**Issue by jamesadevine** [microsoft/jacdac#526]
**Describe the bug**

User feedback: 
 > Accelerometer. I was not able to access this output; I tried shaking the Microbit to see if values change. I however see an unlabeled clickable graphic (I am assuming this is a graph).

**Comment by pelikhan** [comment]
Right, this is going to be the same problem with 3 values changing rapidely.
